spec/controllers/v0/sign_in_controller_spec.rb

```diff
@@ -577,7 +578,11 @@
       end
 
       it 'logs the failed callback' do
-        expect(Rails.logger).to receive(:info).with(expected_error_log, expected_error_message)
+        if sis_standard_error
+          expect(Rails.logger).to receive(:info).with(expected_error_log, expected_error_message)
+        else
+          expect(Rails.logger).to receive(:error).with(expected_error)
+        end
```

Comment:
This kind of conditional expectation implies you should probably set up a new test scenario rather than piggybacking on an existing one. That could be a new shared example or just a new context where you need it. I would definitely avoid doing something like this, though.